src/solver/conv_winoRxS.cpp

```diff
@@ -175,6 +175,93 @@ namespace {
     }
 // clang-format on
 
+// Winograd v30 is functionally supported on Vega10/Vega20 ASICs, but performance regression is
+// expected to be ~25%. Use Winograd v21 instead.
+// Details: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1927#issuecomment-1412741130
+template <int Winodata, int Winofilter>
+inline bool IsWinogradV30Supported(const std::string& asic, const ProblemDescription& problem)
+{
+    return (!StartsWith(asic, "gfx900") && !StartsWith(asic, "gfx906")) ||
+           (IS3X2 && problem.kernel_stride_w == 2);
```

Comment:
Yes, you are right, it allows to use of v30 F(3,2) kernels for `gfx900/gfx906` ASICs in stride2 problems. Winograd v30 F(3,2) stride2 kernels still will have a ~25% performance drop due to [this](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1927#issuecomment-1412741130) on Vega ASICs. However, it can potentially outperform other kernels in some cases.